ci_framework/roles/libvirt_manager/tasks/create_network.yml

```diff
@@ -0,0 +1,46 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Ensure the virtual network is started.
+  block:
+    - name: Virtual network is started.
+      become: true
+      community.libvirt.virt_net:
+        state: active
+        name: "{{ net_name }}"
+  rescue:
+    - name: "Define the virtual network {{ net_name }}"
+      become: true
```

Comment:
ditto - pass `uri: qemu:///system`

Reply:
Sure. The new commit has the necessary changes.